applications/tari_watcher/src/forker.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2024 The Tari Project
+// SPDX-License-Identifier: BSD-3-Clause
+
+use std::{
+    io,
+    net::{IpAddr, Ipv4Addr},
+    path::PathBuf,
+    process::Stdio,
+};
+
+use anyhow::bail;
+use tokio::{
+    fs,
+    process::{Child, Command},
+};
+
+use crate::{
+    config::{ExecutableConfig, InstanceType},
+    port::PortAllocator,
+};
+
+const TARI_L2_VN: InstanceType = InstanceType::TariValidatorNode;
+const MINO_L1_WALLET: InstanceType = InstanceType::MinoTariConsoleWallet;
```

Comment:
nit: rather avoid the extra indirection of a const (i.e. I have to look at what TARI_L2_VN is). An "empty" enum variant is already in essence a const

Reply:
removed